applications/nrf_desktop/README.rst

```diff
@@ -900,6 +908,13 @@ nRF52840 USB Dongle (nrf52840dongle_nrf52840)
         Input data comes from Bluetooth and is retransmitted to USB.
       * |preconfigured_build_types|
 
+nRF52833 USB Dongle (nrf52833dongle_nrf52833) and nRF52820 Dongle (nrf52820dongle_nrf52820)
+      * Both reference designs are meant for the project-specific hardware and are defined in :file:`nrf/boards/arm/nrf52833dongle_nrf52833` and :file:`nrf/boards/arm/nrf52820dongle_nrf52820`, respectively.
+      * The application is configured to act as both mouse and keyboard.
+      * Input data comes from Bluetooth and is retransmitted to USB.
+        The device is Bluetooth central with Nordic's SoftDevice link layer (nrf52833dongle_nrf52833) or Zephyr's software link layer (nrf52820dongle_nrf52820).
```

Comment:
I think 820 dongle can go alone - that will emphasize the usage of OS LL

Reply:
done